File: gareth-application/src/main/java/org/craftsmenlabs/gareth/application/Application.java
package org.craftsmenlabs.gareth.application;

import org.craftsmenlabs.gareth.rest.config.RestConfiguration;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;
import org.springframework.web.cors.CorsConfiguration;

@SpringBootApplication
@Import({CorsConfiguration.class, RestConfiguration.class})
public class Application {
    public static void main(final String[] args) throws Exception {
        SpringApplication.run(Application.class, args);
    }
}

File: gareth-validator-rest/src/main/kotlin/org/craftsmenlabs/gareth/validator/Application.kt
package org.craftsmenlabs.gareth.validator

//import org.craftsmenlabs.gareth.GarethValidatorConfiguration
import org.craftsmenlabs.gareth.GarethValidatorConfiguration
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(GarethValidatorConfiguration::class)
open class Application {
    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(Application::class.java, *args)
        }
    }
}


